lib/HTTP/HPACK.pm6

```diff
@@ -230,6 +230,10 @@ role HTTP::HPACK::Tables {
     has @!dynamic-table;
     has Int $.dynamic-table-limit = 512;
 
+    method set-header-table-size(Int $new-size) {
```

Comment:
I think matching the attribute name in the method name would be better (e.g. `set-dynamic-table-limit`). Perhaps we should also shrink the table if it's already larger than the new size?

Reply:
Fixing...